Bottom: b0fa7143575f17e797f524f24cfe46836f397a33
Top:    62fedb8ce6024185fb84e87db3b57db8176867df
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-02-10 13:37:54 +0000

Avoid clearing pins / vias from polygons... it is already done!










---

diff --git a/src/draw.c b/src/draw.c
index 1ccd1e6..7a506d0 100644
--- a/src/draw.c
+++ b/src/draw.c
@@ -1291,6 +1291,7 @@ ClearPin (PinTypePtr Pin, int Type, int unused)
       AddPart (Pin);
       return;
     }
+#if 0
   /* Clear the area around the pin */
   if (TEST_FLAG (SQUAREFLAG, Pin))
     {
@@ -1313,6 +1314,7 @@ ClearPin (PinTypePtr Pin, int Type, int unused)
     {
       gui->fill_circle (Output.pmGC, Pin->X, Pin->Y, half);
     }
+#endif
   if ((!TEST_FLAG (PINFLAG, Pin) && !PCB->ViaOn)
       || (TEST_FLAG (PINFLAG, Pin) && !PCB->PinOn))
     return;
